docs/runbooks/HPPNotReady.md

```diff
@@ -1,4 +1,4 @@
-<!-- Edited by Jiří Herrmann, 7 Nov 2022 -->
+<!-- Edited by Jiří Herrmann, 8 Nov 2022 -->
 
 # HPPNotReady
 
```

Comment:
Under "Meaning", I recommend reversing the order of the sentences so that it is consistent with the other runbooks.

I recommend changing "an HPP installation" to "HPP".  A user could think that "HPP installation" means that this alert only fires when an installation goes bad. However, the rest of the module indicates that the this alert is generic and covers all the HPP problems.